packages/invoices/src/OutboundShipment/DetailView/modals/OutboundLineEdit.tsx

```diff
@@ -62,11 +62,13 @@ export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
     return true;
   };
 
-  const onAllocate = allocateQuantities(
-    status,
-    draftOutboundLines,
-    setDraftOutboundLines
-  );
+  const onAllocate = (newVal: number, packSize: number | null) => {
+    const newAllocateQuantities = allocateQuantities(
+      status,
+      draftOutboundLines
+    )(newVal, packSize);
+    setDraftOutboundLines(newAllocateQuantities ?? draftOutboundLines);
+  };
```

Comment:
I made the `allocateQuantities` fn just return a new array rather than pass in the setState fn to make testing easier